### ISSUE TITLE
[Todo before demo] Correction list

### ISSUE THREAD

**Issue by Samuelfaure** [etalab/entreprise.data.gouv.fr#209]
Fixed in PR #208 :

Fixed wording for children
Changed wording for capital
Changed immatriculation info order
Improved addresses display
Added a link to moral persons SIREN
Improved observations display

Remaining to fix :

0/ Show address of children siret (Not possible right now)
1/ Remove header (BASE SIRENE, mis à jour le...), replace by maybe a text at the end ?
2/ Aggregate NAME + Surnames (especially in Gestionnaires)
3/ Remove JSON Button / put it less visible at the end of the page.

**Comment by Samuelfaure** [comment]
0/ Is possible right now btw

**Comment by thimy** [comment]
Are we good here?

**Comment by Samuelfaure** [comment]
We havent done 0/ yet but it doesnt need to be done before demo. We good until wednseday id say

**Comment by Samuelfaure** [comment]
Closing here opening new issue
